### ISSUE TITLE
Popovers don't fade under status bar + modal shifts around

### ISSUE THREAD

**Issue by Rohansi** [SteffeyDev/react-native-popover-view#60]
**Describe the bug**
My app is set up to render under a translucent status bar but modals always avoid it. I'm using the `rn-modal` mode and it works alright but I can see the status bar area does not fade and the popover shifts around vertically after animating in sometimes. I suspect this is due to the status bar differences.

I see there is a [`statusBarTranslucent`](https://reactnative.dev/docs/modal#statusbartranslucent-1) prop for `Modal` but there is no way for me to pass that in. I have tried the other modes but they do not work for my use case.

**Device/Setup Info:**
 - Device: Moto e5 cruise, Samsung Galaxy S7 edge, Samsung Galaxy S10+
 - OS: Android 8 to 10
 - `react-native` version: 0.61 (Expo SDK 37)
- `react-native-popover-view` version: 2.0.7

**Debug Output**
```
componentWillReceiveProps - Awaiting popover show
setDefaultDisplayArea - newDisplayArea: {"x":10,"y":34,"width":340,"height":524}
setDefaultDisplayArea - displayAreaOffset: {"x":0,"y":0}
setDefaultDisplayArea (inside calculateRect callback) - fromRect
setDefaultDisplayArea (inside calculateRect callback) - getDisplayArea(): {"x":10,"y":34,"width":340,"height":524}
setDefaultDisplayArea (inside calculateRect callback) - displayAreaStore: {"x":10,"y":10,"width":340,"height":548}
setDefaultDisplayArea (inside calculateRect callback) - Triggering state update
handleGeomChange - requestedContentSize: : {"y":0,"height":290,"width":178.5,"x":0}
computeGeometry - displayArea: {"x":10,"y":34,"width":340,"height":524}
computeGeometry - fromRect: {}
handleGeomChange - Triggering popover move to: {"x":90.75,"y":151}
setDefaultDisplayArea - newDisplayArea: {"x":10,"y":10,"width":340,"height":548}
setDefaultDisplayArea - displayAreaOffset: {"x":0,"y":0}
setDefaultDisplayArea (inside calculateRect callback) - fromRect
setDefaultDisplayArea (inside calculateRect callback) - getDisplayArea(): {"x":10,"y":10,"width":340,"height":548}
setDefaultDisplayArea (inside calculateRect callback) - displayAreaStore: {"x":10,"y":34,"width":340,"height":524}
setDefaultDisplayArea (inside calculateRect callback) - Triggering state update
handleGeomChange - requestedContentSize: : {"y":0,"height":290,"width":178.5,"x":0}
computeGeometry - displayArea: {"x":10,"y":10,"width":340,"height":548}
computeGeometry - fromRect: {}
handleGeomChange - Triggering popover move to: {"x":90.75,"y":139}
measureContent - Showing Popover - requestedContentSize: {"y":0,"height":290,"width":178.5,"x":0}
computeGeometry - displayArea: {"x":10,"y":10,"width":340,"height":548}
computeGeometry - fromRect: {}
measureContent - Showing Popover - geom: {"popoverOrigin":{"x":90.75,"y":139},"anchorPoint":{"x":180,"y":284},"forcedContentSize":{"width":340,"height":548},"viewLargerThanDisplayArea":{"width":false,"height":false},"showArrow":false}
measureContent - Showing Popover - Animating In
getTranslateOrigin - popoverOrigin: {"x":90.75,"y":139}
getTranslateOrigin - popoverSize: {"width":178.5,"height":290}
getTranslateOrigin - anchorPoint: {"x":180,"y":284}
getTranslateOrigin - shift: {"hoizontal":0,"vertical":0}
animateIn - onOpenComplete - Calculated Popover Rect: {"x":91,"y":139,"width":178.5,"height":290}
```
This is the output from when the popover moved after animating in. We can see it started at Y=151 but then moved to Y=139. The status bar height is 24 and this is a difference of 12. This makes sense because the modal is set to be centered.

**Comment by SteffeyDev** [comment]
@Rohansi I just released 2.0.8 and added the `statusBarTranslucent` prop, which simply passes the value through to the underlying `Modal`.  Can you see if that makes a difference?

**Comment by Rohansi** [comment]
@SteffeyDev Thank you so much for the fast response! That should fix the status bar issue - but unfortunately it's React Native 0.62 and up! I'll have to accept that one.

However, the shifting still occurs. I will look into it and submit a pull request for you to review.

**Comment by SteffeyDev** [comment]
Ok, released 2.0.9 with the #61 changes
